### PR TITLE
Kalai linkedlist v1

### PR DESCRIPTION
**Pull request by 31673**
changed format of code

### CODE REVIEW

File: linkedlist.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		printf("%d\n", newNode->data);
+		count ++;
+		newNode = newNode->next;
+	}
+	printf("\nTotal nodes:%d\n", count);
+}
+
+int main(){
+    int choice;
+    void (*ptr [3])()={Insertion_Beginning, Display};
```
Comment (P-Prajwal) [inline linkedlist.c:41]: Why 3 pointers needed for 2 functions?
```suggestion
    void (*ptr [2])()={Insertion_Beginning, Display};

```

File: linkedlist.c [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node{
+	int data;
+	struct node *next;
```
Comment (P-Prajwal) [inline linkedlist.c:6]: Please push error free code, this will give compilation error.
```suggestion
	struct Node *next;

```

File: linkedlist.c [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		count ++;
+		newNode = newNode->next;
+	}
+	printf("\nTotal nodes:%d\n", count);
+}
+
+int main(){
+    int choice;
+    void (*ptr [3])()={Insertion_Beginning, Display};
+    while(1)
```
Comment (tklogesh) [inline linkedlist.c:42]: Conversion related to [PR-2](https://github.com/preeja398/Training/pull/2#discussion_r1036699187), Are you sure, if you give non-integer number as a choice will it work, If so what output you get?

File: linkedlist.c [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    {
+	    printf("\n0.Insertion_Beginning\n1.Display\n");
```
Comment (P-Prajwal) [inline linkedlist.c:44]: Please provide way to exit this loop, its going infinitely. Something like exit choice number.

Example:
If (choice==2)
break;

```suggestion
	    printf("0.Insertion_Beginning\n1.Display\n2.Exit\n");

```

File: linkedlist.c (hunk 1 above)
Comment (31673) [reply linkedlist.c:41]: ok ,I will change

File: linkedlist.c (hunk 3 above)
Comment (31673) [reply linkedlist.c:42]: It will goes infinite loop.

File: linkedlist.c (hunk 4 above)
Comment (31673) [reply linkedlist.c:44]: I added exit option , please verify I changed as you review command.

**Pull request closed without merging**